package.json

```diff
@@ -3,6 +3,8 @@
   "version": "1.0.0-beta.1",
   "description": "Discover, Install, Uninstall, Submit, Remove Adobe App Builder templates",
   "dependencies": {
+    "@adobe/aio-cli-lib-app-config": "^0.2.2",
+    "@adobe/aio-lib-console-project-installation": "github:adobe/aio-lib-console-project-installation#DEVX-2298-APIS",
```

Comment:
TODO: need to update once "adobe/aio-lib-console-project-installation" published on npm